fitnessllm_dataplatform/stream/strava/services/etl_interface.py

```
@@ -0,0 +1,245 @@
+        self.client = bigquery.Client()
+
+    @beartype
+    def load_json_into_bq(self) -> None:
+        try:
+            streams = [
+                StravaStreams[element.name.upper()]
+                for element in self.partial_strava_storage(strava_model=None).iterdir()
+            ]
+        except KeyError as exc:
+            logger.error(f"User defined data_streams for Strava not found: {self.data_streams}: {exc}")
+            raise exc
```

Ruff (0.8.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)